src/CalcViewModel/Common/Utils.cpp

```diff
@@ -50,8 +50,7 @@ double Utils::GetDoubleFromWstring(wstring input)
 {
     wchar_t unWantedChars[] = { L' ', L',', 8234, 8235, 8236, 8237 };
     wstring ws = RemoveUnwantedCharsFromWstring(input, unWantedChars, 6);
-    string inputString(ws.begin(), ws.end());
-    return ::atof(inputString.c_str());
+    return stod(ws);
```

Comment:
We should explicitly use the widechar version of functions.  We're never going to _not_ ship this in Unicode.

```suggestion
    return wcstod(ws);
```

Reply:
stod operates on std::wstring, while wcstod is a C function and it would need to be wcstod(ws.c_str()). I'd prefer to leave it as-is.